### PR TITLE
docs: Add migration documentation from 1.0 to 1.1

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
﻿GitHub Issue: #https://github.com/unoplatform/uno.toolkit.ui/issues/161

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Description

Add migration documentation from Uno.Themes version 1.0 to 1.1

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [ ] Tested Android
- [X] Tested WASM
- [ ] Tested MacOS
- [X] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -362,6 +364,76 @@ Inside your AppTheme add two item's "datePickerDialogTheme" (the style for your
...
+
+	Now you have the possibility to override the Material color palette with your own color palette. See the #Getting Started section for more details.
+	```xml
+	<MaterialColors xmlns="using:Uno.Material"
+					OverrideSource="ms-appx:///ColorPaletteOverride.xaml" />
+	```
+
+- Namespace breaking changes
+
+	BREAKING CHANGE: Everything (controls, extensions, converters,...) is now under `Uno.Material` and under `Uno.Cupertino`.
```
Comment (Xiaoy312) [inline README.md:381]:
```suggestion
	BREAKING CHANGE: Everything (controls, extensions, converters, ...), previously under `Uno.Material.*` or `Uno.Cupertino.*`, has now been moved under `Uno.Material` or `Uno.Cupertino`.
```

File: README.md [hunk 2]
```diff
@@ -199,6 +199,8 @@ For example, if you would like change the `CornerRadius` of all the `Buttons` us
 | CheckBox         | MaterialCheckBoxStyle <br> MaterialSecondaryCheckBoxStyle                         |
 | ComboBox         | MaterialComboBoxStyle                                                             |
 | CommandBar       | MaterialCommandBarStyle                                                           |
+| ListView         | MaterialListViewStyle <br> |
```
Comment (Xiaoy312) [inline README.md:202]:
```suggestion
| ListView         | MaterialListViewStyle <br> MaterialListViewDetailsStyle |
```

File: README.md [hunk 3]
```diff
@@ -199,6 +199,8 @@ For example, if you would like change the `CornerRadius` of all the `Buttons` us
...
+| ListViewItem     | MaterialListViewItemStyle <br> |
```
Comment (Xiaoy312) [inline README.md:203]:
```suggestion
| ListViewItem     | MaterialListViewItemStyle |
```

File: README.md [hunk 4]
```diff
@@ -362,6 +364,76 @@ Inside your AppTheme add two item's "datePickerDialogTheme" (the style for your
...
+	| **Controls**              | **StyleNames**                                                                |
+	|---------------------------|------------------------------------------------------------------------------|
+	| BottomNavigationBar       | MaterialBottomNavigationBarStyle                                              |
+	| ExpandingBottomSheet      | MaterialExpandingBottomSheetStyle                                             |
+	| ModalStandardBottomSheet  | MaterialModalStandardBottomSheetStyle                                         |
+	| StandardBottomSheet       | MaterialStandardBottomSheetStyle                                              |
+	| SnackBar                  | MaterialSnackBarStyle                                                         |
+
+	- BottomNavigationBar was replaced by TabBar in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it is not an exact 1:1 replacement.
+	In the meant time, if you really need the badge and/or other customizabilities, you have two options:
```
Comment (jeromelaban) [inline README.md:412]:
```suggestion
	In the mean time, if you really need the badge and/or other customizability, two options are available:
```

File: README.md [hunk 5]
```diff
@@ -362,6 +364,76 @@ Inside your AppTheme add two item's "datePickerDialogTheme" (the style for your
...
+	Import locally the old sources (control + style) from Uno.Themes; 
+
+		OR
+
+		Copy the MaterialBottomTabBarItemStyle from [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), and modify the style to suit your needs.
+		(Note that there is two copy of it, one for ios and android and one for rest of the platforms: uwp, skia, wasm, etc...)
```
Comment (jeromelaban) [inline README.md:418]: What's "it"? Avoid pronouns if possible.

File: README.md [hunk 6]
```diff
@@ -362,6 +364,76 @@ Inside your AppTheme add two item's "datePickerDialogTheme" (the style for your
...
+                LightDismissOverlayMode="On"
+                FlyoutPresenterStyle="{StaticResource MaterialFlyoutPresenterStyle}">
+			<Grid MaxHeight="370"
+				  VerticalAlignment="Bottom">
+				...Your bottom sheet content...
+			</Grid>
+		</Flyout>
+		```
+
+	- For SnackBar, at the moment we didn't add yet in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui) a control that will replace SnackBar but it's planned.
```
Comment (jeromelaban) [inline README.md:435]:
```suggestion
	- SnackBar is not yet added in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but a control that will replace SnackBar is planned.
```

**Review by jeromelaban — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (davidjohnoliver) [inline README.md:418]:
```suggestion
		(Note that there are two copies of the style, one for iOS and Android and one for rest of the platforms: UWP, Skia, WASM, etc...)
```

File: README.md (hunk 6 above)
Comment (davidjohnoliver) [inline README.md:435]:
```suggestion
	- For SnackBar: so far no replacement for SnackBar has been added to [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it's planned to add one in a future version.
```

**Review by davidjohnoliver — APPROVED** [review]

**Comment by agneszitte-nventive** [comment]
@mergifyio backport release/stable/1.1

**Pull request merged**